platform/viewer/cypress/integration/pwa/OHIFExtensionPDF.spec.js

```diff
@@ -1,23 +1,26 @@
 describe('OHIF PDF Extension', () => {
   before(() => {
     cy.openStudy('Dummy');
-    cy.expectMinimumThumbnails(1);
+    cy.expectMinimumThumbnails(3);
   });
 
   it('checks if series thumbnails are being displayed', () => {
-    cy.get('[data-cy="thumbnail-list"]')
+    cy.get('[data-cy="thumbnail-list"]', { timeout: 15000 })
```

Comment:
If we're using the `cy.expectMinimumThumbnails` in the `before` hook, we may no longer need a timeout for these tests.

Reply:
Hmm. Here the PDF (`DOC`) thumbnails are taking a really long time to load. Either we increase the number of expected thumbnails to `5` or maybe `7`, or we leave the `15s` timeout. Either way, we need to wait more time here.